bundles/config/org.eclipse.smarthome.config.xml.test/src/test/resources/test-bundle-pool/ConfigDescriptionsTest.bundle/ESH-INF/config/config.xml

```diff
@@ -70,7 +70,16 @@
 	            <criteria name="binding-id">hue</criteria>
 	        </filter>
 		</parameter>
+		
+		<parameter name="unit" type="integer" unit="m" />
 
+        <parameter name="unit-label" type="decimal">
+            <unitLabel>Meter</unitLabel>
```

Comment:
this is probably a bad example as imho the label for meters would be "m". So maybe rather use unit=Cel and unitLabel=°C as an example.